packages/lism-ui/src/components/Tooltip/_style.css

```
@@ -119,7 +119,7 @@
       --_spanEnd: span-x-start;
       --_placeStart: auto start;
       --_placeEnd: auto end;
-      margin-block: var(--_gap);
+      margin-block: var(--gap);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)